client/package.json

```diff
@@ -12,8 +12,14 @@
     }
   ],
   "devDependencies": {
+    "babel-core": "^6.10.4",
```

Comment:
this is not needed in `devDependencies` it is loaded from [superdesk](https://github.com/superdesk/superdesk-client-core/blob/v1.5.1/package.json#L45)